sdk/storage/azure-storage-blob/README.md

```diff
@@ -105,6 +108,30 @@ Blob Storage is designed for:
 - Storing data for backup and restore, disaster recovery, and archiving
 - Storing data for analysis by an on-premises or Azure-hosted service
 
+### URL format
+Blobs are addressable using the following URL format:
+The following URL addresses a blob:
+https://myaccount.blob.core.windows.net/mycontainer/myblob
+
+#### Resource URI Syntax
+For the storage account, the base URI for blob operations includes the name of the account only:
+
+```$xslt
+https://myaccount.blob.core.windows.net
+```
+
+For a container, the base URI includes the name of the account and the name of the container:
+
+```$xslt
+https://myaccount.blob.core.windows.net/mycontainer
+```
+
+For a blob, the base URI includes the name of the account, the name of the container and the name of the blob:
+
+```$xslt
+https://myaccount.blob.core.windows.net/mycontainer/myblob
+```
+
```

Comment:
Maybe add a note that CDNs or other specific scenarios my have different formats? That could be too much for the README, though.

Reply:
yeah I'm thinking this is more for people using this for the first time? for more complex scenarios I'm assuming people will know what theyre doing

Reply:
just added a note that this doesnt apply for more advanced scenarios like custom domain names